### PR TITLE
HW1: Parts 1, 2 and 3

### PR DESCRIPTION
**Pull request by Beza4598**

### CODE REVIEW

File: baa2159.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Open Source Development
+
+Name: Beza Amsalu
+
+UNI: baa2159
+
+Github: [link](https://github.com/Beza4598)
+
+Github README: [link](https://github.com/Beza4598/Beza4598/blob/a2c5ac7ac8e7a74e59a333c29794895680303164/README.md)
```
Comment (timkpaine) [inline baa2159.md:9]: use branch link not commit link:
`https://github.com/Beza4598/Beza4598/blob/main/README.md`

File: baa2159.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (timkpaine) [inline baa2159.md:10]: add at least 1 "extra" to your README, like a badge or an embeddable app

File: baa2159.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+## Project Proposal - Spotify to Apple Music Converter
+
+[full project desc](https://github.com/Beza4598/project-proposals-s2023/blob/main/project_descreption.md)
```
Comment (timkpaine) [inline baa2159.md:15]: use relative link like `./projects/python/<your library name>.md`

File: baa2159.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+I want to develop a full-stack program that will convert a playlist from Apple Music to Spotify and vice-versa. 
+
+## Open Source Project Contribution - Matplotlib
+
+[full project desc](https://github.com/Beza4598/project-proposals-s2023/blob/main/project_contribution.md)
```
Comment (timkpaine) [inline baa2159.md:21]: use relative link

File: project_contribution.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+## Matplotlib
```
Comment (timkpaine) [inline project_contribution.md:1]: move to `projects/python/matplotlib.md`

File: project_descreption.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+## Spotify to Apple Music Converter
```
Comment (timkpaine) [inline project_descreption.md:1]: move to `projects/python/spotify_to_apple.md` or whatever you want to call it.

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by Beza4598** [comment]
Resolved all conversations

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**